spec/values/calculation/round/strategy/down.hrx

```diff
@@ -0,0 +1,198 @@
+<===> positive_and_infinity/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, 10, infinity))}
+
+<===> positive_and_infinity/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> negative_and_infinity/input.scss
+@use "sass:math";
+a {b: round(down, -10, infinity)}
+
+<===> negative_and_infinity/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> upper_multiple/number_is_bigger/input.scss
+a {b: round(down, 23px, 10px)}
+
+<===> upper_multiple/number_is_bigger/output.css
+a {
+  b: 20px;
+}
+
+<===>
+================================================================================
+<===> upper_multiple/number_is_smaller/input.scss
+a {b: round(down, 18px, 10px)}
+
+<===> upper_multiple/number_is_smaller/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> upper_multiple/number_is_half/input.scss
+a {b: round(down, 15px, 10px)}
+
+<===> upper_multiple/number_is_half/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> upper_multiple/number_is_negative/input.scss
+a {b: round(down, -13px, 10px)}
+
+<===> upper_multiple/number_is_negative/output.css
+a {
+  b: -20px;
+}
+
+<===>
+================================================================================
+<===> lower_multiple/number_is_bigger/input.scss
+a {b: round(down, 13px, 10px)}
+
+<===> lower_multiple/number_is_bigger/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> lower_multiple/number_is_negative/input.scss
+a {b: round(down, -18px, 10px)}
+
+<===> lower_multiple/number_is_negative/output.css
+a {
+  b: -20px;
+}
+
+<===>
+================================================================================
+<===> negative_zero/positive_infinity/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, -1 * 0, infinity))}
+
+<===> negative_zero/positive_infinity/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> positive_zero/zero/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, 0, infinity))}
+
+<===> positive_zero/zero/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> positive_zero/one/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, 1, infinity))}
+
+<===> positive_zero/one/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> nested_function/input.scss
+a {b: round(down, round(100, 10) + 2 * 3, 10)}
+
+<===> nested_function/output.css
+a {
+  b: 100;
+}
```

Comment:
I don't think this really tests anything more than the simplification logic that's tested elsewhere.